taskfile.yaml

```diff
@@ -365,6 +374,7 @@ tasks:
       - "{{.TASKFILE}}"
       - "client/package.json"
```

Comment:
```
      - "**/{,client,common,server,yscope-log-viewer}/package.json"
      - "**/{,client,common,server,yscope-log-viewer}/package-lock.json"
```

supposedly we can put the `{-lock}` part into the same line but it will require a new version of go-task that contains this fix https://github.com/go-task/task/issues/2073 . Still, semantically, it also makes sense to keep them separate items